src/types.ts

```diff
@@ -38,7 +38,7 @@ export interface ChildrenProps {
 }
 
 export interface GeometryProps {
-    fills?: ReadonlyArray<Paint>;
+    fills?: ReadonlyArray<Paint> | symbol;
```

Comment:
I had to add `symbol` here to make the `geometryMixin` happy. Unfortunately this cascades through to:

https://github.com/react-figma/react-figma/blob/bdb952064f9f7a2b78a2110084e5dfe5d4dfb68d/src/localStyles/useFillPaintStyle.ts#L29-L32

requiring the addition of `symbol` to the `createOrUpdatePaintStyle` args too.